### PR TITLE
GROK-13661: Scripting documentation improve

### PR DESCRIPTION
**Pull request by GennadiyZakharov**
- Column selectors
- FileIO
- Converting script to application

### CODE REVIEW

File: help/compute/scripting-for-non-developers.mdx [hunk 1]
```diff
@@ -297,19 +301,199 @@ When you run the script, you will see the following dialog:
...
+#description: Saves a dataframe to JSON file
+#language: python
+#tags: template, demo, FileIo
+#input: dataframe df [Dataframe to convert to JSON]
+#output: file json_file
+
+df.to_json(json_file)
+```
+
+When you run this script, Datagrok will return to you the `FileInfo` object in the scalar variables panel.
```
Comment (skalkin) [inline help/compute/scripting-for-non-developers.mdx:361]: "to you" is not needed

**Review by skalkin — CHANGES_REQUESTED** [review]
Looks good to me, just some minor comments.

File: help/compute/scripting-for-non-developers.mdx [hunk 2]
```diff
@@ -230,17 +230,21 @@ For additional capabilities of the Datagrok script editor, see the
...
+Also, Datagrok supports `datetime` as a specific parameter type.
+
+For table data, Datagrok supports **Dataframe** as input/output parameters.
 For [Python](https://www.python.org/) this parameter
 is a [Pandas](https://pandas.pydata.org/)
 [dataframe](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html).
 [R](https://www.r-project.org/) supports dataframes natively.
 
-Let's modify it to accept and return both dataframes and scalars.
+Let's modify the default example accept and return both dataframes and scalars.
```
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply help/compute/scripting-for-non-developers.mdx:247]: Fixed

File: help/compute/scripting-for-non-developers.mdx [hunk 3]
```diff
@@ -297,19 +301,199 @@ When you run the script, you will see the following dialog:
 ![DataframeIdDemo input](pics/dataframeiddemo-input.png)
 
 Datagrok created the script UI,
-populated default values, and created popups with help text
+populated default values, and created popups with help text.
 
-After running this script, Datagrok automatically opens the new dataframe.
+After running this script, 
+Datagrok automatically catches and opens the new dataframe.
```
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply help/compute/scripting-for-non-developers.mdx:307]: Agree. Fixed.

File: help/compute/scripting-for-non-developers.mdx [hunk 4]
```diff
@@ -297,19 +301,199 @@ When you run the script, you will see the following dialog:
...
 It will contain an additional column **ID** with the generated row ID.
 
-### Enhancing script input view
+### Using column inputs
+
+For the dataframe, Datagrok provides you special data input to select one or multiple columns.
+* The `column` input parameter allows you to select one column from the dataframe. 
+  In the script it will be a string variable containing column ID
```
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply help/compute/scripting-for-non-developers.mdx:314]: Fixed.

File: help/compute/scripting-for-non-developers.mdx (hunk 1 above)
Comment (GennadiyZakharov) [reply help/compute/scripting-for-non-developers.mdx:361]: Fixed.

File: help/compute/scripting-for-non-developers.mdx [hunk 5]
```diff
@@ -297,19 +301,199 @@ When you run the script, you will see the following dialog:
...
+* The `column_list` input parameter allows you to select multiple columns.
+  In the script it will be a list of strings, containing column IDs
+
+Both of these selectors require at least one **Dataframe** input
```
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply help/compute/scripting-for-non-developers.mdx:318]: I reworked it - I hope now it's better.

**Review by mistressxalexis — COMMENTED** [review]
contextual information

**Review by mistressxalexis — APPROVED** [review]

**Pull request merged**